recipes/macfsevents/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "MacFSEvents" %}
+{% set version = "0.7" %}
+{% set sha256 = "95d3cddaf8a42435bfbd50087785ee9e3ebb8325242cfd06a88ea21f85bcb56f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - fsevents
+
+about:
+  home: https://github.com/malthe/macfsevents
+  license: BSD-2-Clause
+  license_family: BSD
+  # License is available in upstream repo, but not in PyPi Wheel.
+  # Manually packaging it per https://conda-forge.org/docs/meta.html#packaging-the-license-manually
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  summary: Thread-based interface to file system observation primitives.
+  description: "MacFSEvents is a Python library that provides thread-safe directory observation primitives using callbacks. It wraps the Mac OS X FSEvents API in a C-extension.
+
+Requirements:
+
+Mac OS X 10.5+ (Leopard)
+Python 2.7+
+
+This software was written by Malthe Borch <mborch@gmail.com>. The pyfsevents module by Nicolas Dumazet was used for reference."
```

Comment:
nit: Would be nice if the `description` section were a bit more contained. Recommend using `description: |` and indenting text to fit underneath. This should improve readability. Here's a [good example]( https://github.com/conda-forge/willow-feedstock/blob/6add334d5a8d297b0b078b5f2433ce6a5de17f28/recipe/meta.yaml#L35-L38 ).